shifthappens/tasks/__init__.py

```diff
@@ -1 +1,3 @@
 from .base import Task  # noqa: F401
+
+from shifthappens.tasks import imagenet_m
```

Comment:
```suggestion
from shifthappens.tasks import imagenet_m  # noqa: F401
```
Fix to pass mypy test.